packages/presentation/src/components/UsersPopup.svelte

```diff
@@ -34,14 +34,24 @@
   export let placeholder: IntlString = presentation.string.Search
   export let selectedUsers: Ref<Person>[] = []
   export let ignoreUsers: Ref<Person>[] = []
+  export let shadows: boolean = true
 
   let search: string = ''
   let objects: Person[] = []
   let input: HTMLInputElement
 
   const dispatch = createEventDispatcher()
   const query = createQuery()
-  $: query.query<Person>(_class, { name: { $like: '%' + search + '%' }, _id: { $nin: ignoreUsers } }, result => { objects = result }, { limit: 200 })
+  $: if (search.length)
```

Comment:
If you really need it please add any export boolean flag for it, but I think it's not necessary. Currently behavior has been broken elsewhere , you can see it in CI uitests results